### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by rodrigoprimo**
[MAILPOET-5232]

[MAILPOET-5232]: https://mailpoet.atlassian.net/browse/MAILPOET-5232?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/lib/Segments/DynamicSegments/SegmentSaveController.php [hunk 1]
```diff
@@ -31,8 +31,8 @@ public function __construct(
    */
   public function save(array $data = []): SegmentEntity {
     $id = isset($data['id']) ? (int)$data['id'] : null;
-    $name = isset($data['name']) ? sanitize_text_field($data['name']) : '';
-    $description = isset($data['description']) ? sanitize_textarea_field($data['description']) : '';
+    $name = $data['name'] ?? '';
+    $description = $data['description'] ?? '';
```
Comment (triple0t) [inline mailpoet/lib/Segments/DynamicSegments/SegmentSaveController.php:35]: If we are doing this here, You might also want to update this section: https://github.com/mailpoet/mailpoet/blob/0bbd65ac61f009ea234ddf59baf5b10e507ca11d/mailpoet/lib/API/MP/v1/Segments.php#L45-L46

**Review by triple0t — COMMENTED** [review]
LGTM

File: mailpoet/lib/Segments/DynamicSegments/SegmentSaveController.php (hunk 1 above)
Comment (rodrigoprimo) [reply mailpoet/lib/Segments/DynamicSegments/SegmentSaveController.php:35]: Thanks for the review, @triple0t. I will follow on a DM.

**Review by brezocordero — APPROVED** [review]
The changes look good.

**Pull request merged**